packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -711,6 +728,15 @@ export class AppManagementClient implements DeveloperPlatformClient {
       variables,
     )
   }
+
+  private async organizationBetaFlags(_organizationId: string, allBetaFlags: string[]): Promise<{[key: string]: boolean}> {
```

Comment:
Are the flags here and in the github manifest file going to be hashed? There needs to be a strong reason if not.

Reply:
We could generate flags with obfuscated names... in public PRs explaining everything 😅